app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+
+import { Button, Toastr } from "@bigbinary/neetoui";
+import { Container, Header, Scrollable } from "@bigbinary/neetoui/layouts";
+import { useTranslation } from "react-i18next";
+
+import DeleteAlert from "components/commons/DeleteAlert";
+
+import { TABLE_ROW_DATA, MENU_BAR_OPTIONS } from "./constants";
+import Table from "./Table";
+
+import MenuBar from "../../commons/MenuBar";
+
+const Contacts = () => {
+  const [showMenu, setShowMenu] = useState(false);
+  const [searchString, setSearchString] = useState("");
+  const [contacts, setContacts] = useState(TABLE_ROW_DATA);
+  const [showDeleteAlert, setShowDeleteAlert] = useState(false);
+  const [selectedContactId, setSelectedContactId] = useState(null);
+
+  const { t } = useTranslation();
+
+  const deleteContact = () => {
+    const nonDeletedNotes = contacts.filter(
+      contact => contact.id !== selectedContactId
+    );
+    setContacts(nonDeletedNotes);
+    Toastr.success(t("contact.delete.success"));
+  };
+
+  const handleDelete = id => {
+    setSelectedContactId(id);
+    setShowDeleteAlert(true);
+  };
+
+  return (
+    <div className="max-w-fit flex">
+      <MenuBar menuBarOptions={MENU_BAR_OPTIONS} showMenu={showMenu} />
+      <Container>
+        <Header
+          menuBarToggle={() => setShowMenu(!showMenu)}
```

Comment:
https://academy.bigbinary.com/learn-react/react-best-practices/know-when-to-use-previous-state-for-state-update